opm/input/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -139,6 +141,12 @@ namespace Opm {
         , m_micppara(          deck)
         , wag_hyst_config(     deck)
     {
+        if (deck.hasKeyword("GRIDOPTS")) {
+            const auto& gridOpts = deck["GRIDOPTS"].back();
+            const auto& record = gridOpts.getRecord(0);
+            m_write_all_multminus = record.getItem("TRANMULT").get<std::string>(0)== "YES";
```

Comment:
Just to be on the safe side, we should probably use `getTrimmedString(0)` instead of `get<std::string>(0)` here.

Reply:
thanks. Amended.